python-checks/src/test/resources/checks/className.py

```diff
@@ -1,8 +1,8 @@
 class MyClassWithCompliantName1:
 	pass
 
-class MyClass_WithNotCompliantName1:
+class MyClass_WithNotCompliantName1: # Noncompliant {{Rename class "MyClass_WithNotCompliantName1" to match the regular expression ^[A-Z_][a-zA-Z0-9]+$.}}
```

Comment:
@vilchik-elena We should assert the precise location: that's what the ticket is about...